modern/src/resources/l10n/en.json

```diff
@@ -1,594 +1,595 @@
 {
-    "sharedLoading": "Loading...",
```

Comment:
Something happened with the localization file.

Reply:
I didn't understand why it was formatted like that, I flew or corrected it.